common/cmd_nvedit.c

```diff
@@ -203,6 +204,16 @@ int _do_setenv (int flag, int argc, char *argv[])
 		}
 #endif
 
+		if (strcmp(argv[1],"bootflag") == 0) {
+			ulong flag = simple_strtoul(argv[2], NULL, 16);
+			if (0x5A5A5A5A != flag && 0xA5A5A5A5 != flag) {
+				printf("wrong flag, the bootflag can only be either 0x5A5A5A5A (for #img A) or 0xA5A5A5A5 (for #img B)\n");
+				return 1;
```

Comment:
Should we add `sys_bootflag = BOOT_FROM_A` here?

Reply:
please check the latest commit on this PR